apps/nextra/pages/en/build/cli/use-cli/running-a-local-network.mdx

```diff
@@ -0,0 +1,188 @@
+import { Callout, Steps } from 'nextra/components'
+
+# Running a Local Network via Aptos CLI
+
+Local networks can be helpful when testing your code. They are not connected to any production Aptos networks like mainnet, but they are useful for three main reasons:
+
+1. **No rate limits:** You can interact with hosted services like the Node API, Indexer API, and faucet with no rate-limits to speed up testing.
+2. **Reproducibility:** You can set up specific on-chain scenarios and restart the network from scratch at any point to return to a clean slate.
+3. **High availability**: The Aptos devnet and testnet networks are periodically upgraded, during which time they can be unavailable. Local development networks are also always available even if you have no internet access.
+
```

Comment:
Can we add a `<br />` here. Feels like it needs a bit of space (https://developer-docs-nextra-git-fork-jmintuitive-mig-294fb4-aptoslabs.vercel.app/en/build/cli/use-cli/running-a-local-network)

Reply:
Agreed, looks much better :)